nuxeo-core/nuxeo-core-mimetype/src/main/java/org/nuxeo/ecm/platform/mimetype/NXMimeType.java

```diff
@@ -37,6 +38,6 @@ private NXMimeType() {
     }
 
     public static MimetypeRegistryService getMimetypeRegistryService() {
-        return (MimetypeRegistryService) Framework.getRuntime().getComponent(MimetypeRegistryService.NAME);
+        return (MimetypeRegistryService) Framework.getService(MimetypeRegistry.class);
```

Comment:
The whole class is deprecated since 7.2, I think you can remove it in a separate commit rather than editing it.
You can commit with [NXP-26772](https://jira.nuxeo.com/browse/NXP-26772).